examples/checkout/search/search_abandoned_transactions.rb

```diff
@@ -8,7 +8,10 @@
 #   You can pass the credentials parameters to PagSeguro::Transaction#find_abandoned
 
 options = {
-  credentials: PagSeguro::AccountCredentials.new('EMAIL', 'TOKEN')
+  credentials: PagSeguro::AccountCredentials.new('EMAIL', 'TOKEN'),
+  starts_at: (Time.now - 86400),
```

Comment:
In the code it can be a good ideia (I mean it will be more readable :D ), but here can confuse "people" :| 

:clap: :+1: